src/struct_ls/pfmg_setup.c

```diff
@@ -377,17 +378,12 @@ hypre_PFMGSetup( void               *pfmg_vdata,
 
    if ((pfmg_data -> logging) > 0)
    {
-      max_iter = (pfmg_data -> max_iter);
       (pfmg_data -> norms)     = hypre_TAlloc(HYPRE_Real, max_iter+1, HYPRE_MEMORY_HOST);
       (pfmg_data -> rel_norms) = hypre_TAlloc(HYPRE_Real, max_iter+1, HYPRE_MEMORY_HOST);
    }
 
-#if DEBUG
+   if (print_level > 1)
```

Comment:
I'm not sure we should use `print_level` to print to file.  We defined the meaning of this at one point, but maybe it's not documented well.  I will need to look this up.

Reply:
Hi Rob, I found the meaning of `print_level` is documented for some `ParCSR` solvers but not for `Struct` or `SStruct`. My reasoning was:
```
0: No printing.
1: Print convergence data.
2: Print MG matrices and vectors to file.
```
With the option 2, I eliminate the need of redefining the DEBUG variable every time I want to see that information. But having to do this is not a big deal either, so let me know if you prefer the way it was before and I'm happy to change.